docs/Installing-an-application-from-APK.md

```diff
@@ -0,0 +1,37 @@
+# Installing an application from APK
+
+## Introduction
+
+This guide will show you how to install an application from an APK file on your Android emulator.
```

Comment:
This guide should also include installing apk on real devices and not just emulators.

Reply:
Updated the description